NXP/MCIMX8M_EVK_4GB/Library/MsPlatformDevicesLib/MsPlatformDevicesLib.c

```diff
@@ -0,0 +1,284 @@
+/*++
+
+    Copyright (C) 2019 Microsoft Corporation. All Rights Reserved.
+
```

Comment:
Lots of this looks like boilerplate code.  Can it be made imx8m generic, rather than board specific?  We'd like to have the minimal amount of board specific files.  Both in quantity and size.  The fact that it starts with MsPlatform is a dead giveaway it might be shared code.